libraries/SITL/SIM_MicroStrain.cpp

```diff
@@ -205,7 +205,84 @@ void MicroStrain5::send_gnss_packet(void)
     send_packet(packet);
 }
 
-void MicroStrain5::send_filter_packet(void)
+void MicroStrain7::send_gnss_packet(void)
```

Comment:
Please put into a `SIM_Microstrain7.cpp`

Reply:
Done. Did the same for the 5 series specific code, and then had to expose `simulation_timeval` in the header.

Reply:
> Done. Did the same for the 5 series specific code, and then had to expose `simulation_timeval` in the header.

If I thought moving the 5-series code as part of this PR was a good idea, I would have asked for that.

What you've done now is added several large deltas that don't need to be in here....

Please rebase this branch and move it out of draft....